build.gradle

```diff
@@ -1,37 +1,63 @@
-//------------------------------------------- PLUGIN DEFINITION -----------------------------------------------//
-
```

Comment:
Can you add back the comments in this file? I want to use this as a template to explain it to the students.